src/inmanta/moduletool.py

```diff
@@ -1074,7 +1131,10 @@ def _do_update(self, output_directory: str, setup_cfg: ConfigParser, warn_on_mer
                 + "\n"
             )
 
-    def get_pyproject(self, in_folder: str, warn_on_merge: bool = False) -> str:
+    @classmethod
+    def get_pyproject(
+        cls, in_folder: str, warn_on_merge: bool = False, build_requires: list[str] = ["setuptools", "wheel"]
```

Comment:
```suggestion
        cls, in_folder: str, warn_on_merge: bool = False, build_requires: Optional[list[str]] = None
```
Nice with the built-in `list`!

Reply:
but then we break the api. Should I set the default inside the method then?